src/app/impac.config.coffee

```diff
@@ -9,9 +9,6 @@ angular.module 'frontendAdmin'
   data =
     mnoHub: mnoHub
     impacApi: "#{IMPAC_CONFIG.protocol}://#{IMPAC_CONFIG.host}/api"
-    dashboards:
```

Comment:
Yes dashboard designer was introduced in `1.0`. As part of this PR, I'm fixing some impac-config and bumping impac-angular to `1.8`

As mentioned elsewhere, I'm planning to tag a new minor version of mnoe (`3.4`) and mnoe-admin-panel (`1.1`) from the `master` branch. 
There was an issue with the branching model when we started the APIv2 (lots of project were actually pointing directly to `master` instead of `3.3`/`1.0`), so we started `4.0`/`2.0` and stopped using the `master` branch until project were migrated.